pyproject.toml

```diff
@@ -174,4 +174,3 @@ max-args = 10
 
 [tool.mypy]
 ignore_missing_imports = true
-python_version = 3.8
```

Comment:
Why this has been removed?

Reply:
Removed it for testing, forgot to add it back. It's back now